src/couch_views/src/couch_views.erl

```diff
@@ -73,6 +74,43 @@ build_indices(#{} = Db, DDocs) when is_list(DDocs) ->
         end
     end, DDocs).
 
+get_info(Db, DDoc) ->
```

Comment:
Getting into the super nitty gritty on style, but its two empty lines between functions.